test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/T4/ODataT4CamelCaseTests.cs

```diff
@@ -465,7 +472,7 @@ public void WriteBodyOperationParametersShouldUseOriginalName()
         internal EntryValueMaterializationPolicy CreateEntryMaterializationPolicy(TestMaterializerContext materializerContext = null)
         {
             var clientEdmModel = new ClientEdmModel(ODataProtocolVersion.V4);
-            var context = new DataServiceContext();
+            var context = new DataServiceContext().ReConfigureForNetworkLoadingTests();

```

Comment:
Do you need to do this here if you're calling `Context.ReConfigureForNetworkLoadingTests();`?

Reply:
This is the only occurrence in the file of creating the context